src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -254,7 +254,7 @@
 
   {% set class = extraVars.class|default('') %}
 
-  <div class="form-group row {{ class }}">
+  <div class="form-group row {{ class }}" {% if extraVars.order is defined %} style="order: {{ extraVars.order }}; -webkit-order: {{ extraVars.order }}" {% endif %}>
```

Comment:
css in html - unfortunately  this part is unavoidable. 

Another option could be:
Unless we can create classes for each order posiiton but we must know how many elements exist in the first place. We can generate **100** classes - but again if we hhave more then 100 elements then its a problem with this functionality.